contracts/SportMarkets/Rundown/TherundownConsumer.sol

```diff
@@ -615,10 +627,15 @@ contract TherundownConsumer is Initializable, ProxyOwned, ProxyPausable {
         address _market,
         uint _outcome,
         uint8 _homeScore,
-        uint8 _awayScore
+        uint8 _awayScore,
+        uint8 _status
     ) internal {
         sportsManager.resolveMarket(_market, _outcome);
-        oddsObtainer.resolveChildMarkets(_market, _outcome, _homeScore, _awayScore);
+        if (_status == 201) {
```

Comment:
Perhaps extract `201` into a variable that indicates what the status is for, e.g. "STATUS_RETIRED"